saleor/graphql/attribute/utils.py

```diff
@@ -148,7 +148,7 @@ def _pre_save_values(
         if not attr_values.values:
             return tuple()
         for value in attr_values.values:
-            value_obj = attribute.values.filter(name=value).first()
+            value_obj = attribute.values.filter(Q(slug=value) | Q(name=value)).first()
```

Comment:
I think we should fetch all values and prefer the one where the slug matches, consider two values:
* `name: "red", slug: "red-2"`
* `name: "RED", slug: "red"`

It will randomly pick one of them every time I save.